sklearn/linear_model/ridge.py

```diff
@@ -111,7 +111,7 @@ def _solve_cholesky(X, y, alpha):
         return linalg.solve(A, Xy, sym_pos=True,
```

Comment:
I double check it, it does passes through. `A` and `Xy` are float32 and float64 (as expected) and `linalg.solve` preserves the type. 

So, all good. And both branches are tested.